### PR TITLE
Implement debug handler

### PR DESCRIPTION
**Pull request by hakonhall**

### CODE REVIEW

File: node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/provider/DebugHandlerHelper.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+package com.yahoo.vespa.hosted.node.admin.provider;
+
+import javax.annotation.concurrent.ThreadSafe;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+
```
Comment (bratseth) [inline node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/provider/DebugHandlerHelper.java]: Please add @author and an explanation of what this is.

Comment (bratseth) [reply node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/provider/DebugHandlerHelper.java]: Same with all the other files.

**Review by freva — APPROVED** [review]
Shouldn't author be `hakonhall` though?

File: node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/provider/DebugHandlerHelper.java (hunk 1 above)
Comment (hakonhall) [reply node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/provider/DebugHandlerHelper.java]: Done

@freva fixed

**Pull request merged**